### PR TITLE
add "Getting Started" page to dlang.org

### PR DESCRIPTION
**Pull request by zachthemystic**
https://issues.dlang.org/show_bug.cgi?id=14088

gettingstartedpage

### CODE REVIEW

**Comment by CyberShadow** [comment]
win32.mak too please.

OT: why is your screenshot so brighter than the site?

**Comment by zachthemystic** [comment]
Don't know. I used the built-in Mac partial screenshot feature "CMD-SHIFT-4". I don't have much experience. I can't test the win32.mak, but I did what I could.

**Comment by CyberShadow** [comment]
> I can't test the win32.mak, but I did what I could.

LGTM.

Is there an existing discussion regarding the proposal of adding this page? If not, maybe a forum thread would be appropriate.

**Comment by zachthemystic** [comment]
http://forum.dlang.org/post/omklmlntzphuwbibxcan@forum.dlang.org

**Comment by quickfur** [comment]
This looks good! We absolutely need something like this. I would even put the "Getting started" link on the left navbar before the changelog link. Nobody cares about the changelog unless they're already invested in the language; what new visitors want to know is how they can try out this interesting new language so that they can decide whether or not the other stuff (like changelogs) are worth their time.

**Comment by zachthemystic** [comment]
@quickfur I was thinking incrementally!!! My next suggestion was to just remove the changelog and keep the link to it on the Download screen. But one thing at a time!

**Comment by CyberShadow** [comment]
> Nobody cares about the changelog unless they're already invested in the language; what new visitors want to know is how they can try out this interesting new language so that they can decide whether or not the other stuff (like changelogs) are worth their time.

I disagree! For people who use D but do not follow the forums, or are simply curious about D's development, the changelog is highly interesting.

I just checked - changelog.html is the number 4 most visited page on the website. Let's leave it in the sidebar, please.

**Comment by quickfur** [comment]
I didn't mean to remove the changelog entirely, just to put the "Getting started" link above it.

**Comment by CyberShadow** [comment]
> just to put the "Getting started" link above it.

OK, I don't disagree with that.

**Comment by zachthemystic** [comment]
Okay, I put Getting Started first.

**Comment by zachthemystic** [comment]
Ok, I put Getting Started above Changelog now.

**Comment by zachthemystic** [comment]
Okay, I figured out how to squash some commits. zachgit += 1.

**Comment by quickfur** [comment]
Looks good to me!

ping @andralex

File: posix.mak [hunk 1]
```diff
@@ -143,8 +143,8 @@ PAGES_ROOT=$(SPEC_ROOT) 32-64-portability acknowledgements ascii-table		\
 	bugstats.php builtin changelog code_coverage concepts const-faq COM	\
 	comparison cpptod ctod D1toD2 d-array-article d-floating-point		\
 	deprecate dll dll-linux dmd-freebsd dmd-linux dmd-osx dmd-windows	\
-	download dstyle exception-safe faq features2 glossary gsoc2011 gsoc2012	\
-	gsoc2012-template hijack howto-promote htod htomodule index intro	\
+	download dstyle exception-safe faq features2 getstarted glossary gsoc2011 \
+	gsoc2012 gsoc2012-template hijack howto-promote htod htomodule index intro \
```
Comment (andralex) [inline posix.mak]: The gsoc2012-template doesn't need to be part of the built files.

**Comment by andralex** [comment]
lgtm modulo that one nit. Thanks!

File: posix.mak (hunk 1 above)
Comment (CyberShadow) [inline posix.mak]: This is not an addition, `gsoc2012` just got shifted to a different line.

**Comment by CyberShadow** [comment]
I'm going to merge this since the nit does not apply, thanks :)

**Comment by zachthemystic** [comment]
Thanks. Btw, @CyberShadow :  I couldn't find the code for the look and feel of the Wiki page in: https://issues.dlang.org/show_bug.cgi?id=14115, which I wanted to try editing.

**Pull request merged**